tests/phpunit/tests/functions.php

```diff
@@ -1224,7 +1224,7 @@ public function test_wp_ext2type() {
 	 */
 	public function test_wp_raise_memory_limit() {
 		if ( -1 !== WP_MAX_MEMORY_LIMIT ) {
```

Comment:
This smells wrong. This is something which should be sorted out via the test set up, not something which should fail the tests.